### PR TITLE
PLATEXP-1703: Updated add-new-vpn-user-myrackspace-portal

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/PLATEXP-1703

@ssalinas24 , please review the article

### CODE REVIEW

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 1]
```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
...
+last_modified_date: '2022-10-14'
+last_modified_by: Asmita Nakwa
 product: Dedicated Hosting
 product_url: dedicated-hosting
 ---
 
 Adding new client VPN users is now an automated task within the MyRackspace
-Portal. This article describes how to add a new VPN user by using a ticket
-template.
+Portal. This article describes how to add a new VPN user by using a Support Center.
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:15]:
```suggestion
Portal. This article describes how to add a new VPN user by using our Support Center.
```

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 2]
```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
...
 
-3. Select **Tickets > Create New Ticket**.
+**Step 2.** In the top navigation bar, click **Support > Support Center**.
 
-4. On the **Create New Ticket** page, click the **Subject** field, and in
-   the drop-down menu, select **VPN User Management - add, reset, delete VPN user(s) on firewall(s)**.
+   <img width="941" alt="Select support center from the top navigation bar" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn2.png">
 
-   {{<image src="image001.jpeg" alt="" title="">}}
+**Step 3.** Under **Common Request**, click **See All Templates**. The See All Templates displays list of all templates.
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:28]:
```suggestion
**Step 3.** Under **Common Request**, click **See All Templates**. A list of all Common Request templates will be displayed.
```

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 3]
```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
...
 
-6. In the **Firewall** drop-down field, select the appropriate firewall(s).
+**Step 4.** From the **Common Request** menu, select **VPN User Management - add, reset, delete VPN user(s) on firewall(s)**.
 
-   a. If you have high availability (HA) firewalls, you may select only one of the two
-      HA firewalls in the drop-down menu. Automation adds the user to both HA firewalls.
+   <img width="830" alt="Select VPN User Management- add, reset, delete VPN user(s) on firewall(s) from the menu" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn4.png">
 
-   b. If you need to add the VPN user to all firewalls on the account, ensure
+**Step 5.** In **Issue Details** sections you can select one, multiple or all of the following VPN user management options as shown in following image.
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:36]:
```suggestion
**Step 5.** In the **Issue Details** section you can select one, multiple or all of the following VPN user management options as shown in following image.
```

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 4]
```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
...
+
+   <img width="633" alt="Select any one or multiple VPN user on firewall" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn5.png">
+
+**Step 6.** Check the box next to **Create new VPN user on firewall**.
+
+   <img width="408" alt="Tick the option create new VPN" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn6.png">
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:42]:
```suggestion
   <img width="408" alt="Select the create new VPN option" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn6.png">
```

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 5]
```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
...
    the MyRackspace Portal. Automation runs in the background while the ticket
    generates, which typically takes less than a minute to complete. After you
    refresh your browser, the ticket, *Create New VPN User on Firewall Request (from template)*,
    is in **Confirm Solved** status.
 
-10. Retrieve the password for your new VPN user by opening the ticket. The
+**Step 11.** Retrieve the password for your new VPN user by opening the ticket. The
     password is in the most recent comment.
 
    {{<image src="image002.jpeg" alt="" title="">}}
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:81]: I was about to merge, but it seems this image needs to be updated. Looks like an old image that doesn't relate to this step. Please review and fix.
I applied commits to this PR. So you will need to checkout this PR branch on your local and do git pull first. then make your commits and push back up again.

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md (hunk 5 above)
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:81]: Replaced the image with the new one.

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 6]
```diff
@@ -5,66 +5,80 @@ title: Add a new VPN user in the MyRackspace Portal
 type: article
 created_date: '2016-11-10'
 created_by: Trevor Becker
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2022-10-17'
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:8]:
```suggestion
last_modified_date: '2022-10-18'
```

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md [hunk 7]
```diff
@@ -5,66 +5,80 @@ title: Add a new VPN user in the MyRackspace Portal
...
+
+   <img width="633" alt="Select any one or multiple VPN user on firewall" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn5.png">
+
+**Step 6.** Check the box next to **Create new VPN user on firewall**.
+
+   <img width="408" alt="Select the create new VPN option" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn6.png">
+
+**Step 7.** In the **Firewall** drop-down field, select the appropriate firewall(s).
+
+   <img width="370" alt="Select required firewall from the drop-down list" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn7.png">
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:46]: As we noticed, looks like device is showing in image.

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md (hunk 7 above)
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md:46]: Image is replaced as suggested

**Review by ssalinas24 — APPROVED** [review]
LGTM

**Pull request merged**